recipe/meta.yaml

```diff
@@ -28,22 +27,16 @@ requirements:
     - numpy
     - swig
     - hdf5  # [not win]
+    
     - libitk-devel
-    - libitk <5.2  # [win]
-    - libitk  # [not win]
-    - root_base  # [linux]
+    - libitk
     - nlohmann_json
-    - openmp  # [osx]
-    - libgomp  # [linux]
+
+    - root_base  # [linux]
     - zlib  # [win]
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - {{ pin_compatible('libitk', min_pin='x.x', max_pin='x.x') }}
-    - {{ pin_compatible('hdf5', min_pin='x.x', max_pin='x.x') }}  # [not win]
```

Comment:
The thing is, that some of these packages are "pinning" to stricter versions.

* `hdf5` https://github.com/conda-forge/hdf5-feedstock/blob/main/recipe/meta.yaml#L88  likes to pin to `x.x.x`
* `libitk` https://github.com/conda-forge/libitk-feedstock/blob/main/recipe/meta.yaml#L13 pins to `x.x`

Its better to delegate this kind of pinning to the package you depend on, rather than to have to manage it yourself.

Reply:
As for #35. a matching PR was sent to `libitk` to ensure that this problem is resolved in the long term
https://github.com/conda-forge/libitk-feedstock/pull/65

However, even I am scared of relying on these rapid changes, since they are difficult to propagate instantly.

Today, in Dec 2022, it is no longer necessary due to the correct pinnings in libitk.